etl/steps/data/meadow/papers/2022-11-04/riley_2005/riley_2005.text_expected.txt

```diff
@@ -0,0 +1,28 @@
+TABLE 1 Estimates of life expectancy before the beginning of the health transition, by region, and of regional and global 
```

Comment:
I totally see that it would be hard to parse the table programmatically, and it's not worth it. But I don't understand the point of this file, if you are not extracting data from it. It would suffice to check the md5 of the file. Maybe I'm missing something.

Reply:
That makes sense, thanks!